news/2023/2023-12-12-how-advances-in-technology-affected-the-osu-community.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: How advances in technology affected the osu! community
+date: 2023-12-12 12:30:00 +0000
+---
+
+Let's walk back through the past 15 years of osu!, and how various new technologies affected the game and the community.
+
+![](/wiki/shared/news/2023-12-12-how-advances-in-technology-affected-the-osu-community/banner.jpg)
+
+Every now and then there's a new piece of tech that's slated to change the way games are played, whether it be a new input device, a new way to squeeze more FPS out of your PC, or a whole new way to experience games.
+
+While the osu! community has managed to stay unswayed as innovations such as virtual reality and augmented reality, cloud gaming & AI advancements cropped up due to them being inapplicable to osu!, there were still some that changed the way the game is played, or at least challenged the views on the most optimal way to play it.
+
+It's time to dive in and see how technology can impact the community, what the results of widespread adoption are, or could have been.
```

Comment:
```suggestion
It's time to dive in the history lane and see how technology has impacted the community, what the results of widespread adoption are, or could have been.
```

better aligns with what the newspost is doing

Reply:
im not sure you dive down a lane, maybe drive down?

Reply:
drive down works

Reply:
https://github.com/ppy/osu-wiki/pull/10659/commits/df739b94b9237a643ca2bc0ab08974b4ac432a3c i rewrote the entire sentence [as it felt redundant to me](https://discord.com/channels/188630481301012481/218677502141399041/1187169280463212545)